src/app/shared/utils/fields.utils.ts

```diff
@@ -122,4 +122,35 @@ export class FieldsUtils {
     return Object.assign({}, obj);
   }
 
+  showGrayBar(caseField: CaseField, element: HTMLElement) {
+    if (caseField.field_type.type !== 'Collection') {
+      let divSelector = element.querySelector('div');
+      let samePage = true;
+      let cyaSelector = false;
+      this.getConditionFields(caseField.show_condition).forEach(
+        field => { samePage = document.querySelector('#' + field) ? true : false; }
+      );
+      if (divSelector) {
+        let tempElement = divSelector.parentElement;
```

Comment:
I think using dom navigation makes this method a bit fragile. It's not ideal. Let see if we can think together of a better solution. 
I'll give it a thought